json-endpoints/jetpack/class.jetpack-json-api-get-post-backup-endpoint.php

```diff
@@ -3,7 +3,7 @@
 class Jetpack_JSON_API_Get_Post_Backup_Endpoint extends Jetpack_JSON_API_Endpoint {
 	// /sites/%s/posts/%d/backup      -> $blog_id, $post_id
 
-	protected $needed_capabilities = 'read_private_posts';
+	protected $needed_capabilities = array();
```

Comment:
Yes, it's so that we can talk to sites in a user-agnostic way. This is a spanking new endpoint, which is not in use anywhere at this point.